Makefile

```diff
@@ -24,14 +24,14 @@ clean-pyc:
 	find . -name '*~' -exec rm -f {} +
 
 lint:
-	tox -e lint
+	tox -elint
```

Comment:
Just curious, was there a reason for this change? It seems it still works as `tox -elint` but the usage docs examples separate the environment from the `-e`.

Reply:
Oops, nope. Not sure where this change came from.